src/test/it/cdkata.yml

```diff
@@ -0,0 +1,157 @@
+---
+packaging:
+#  bom: bom.yml
+  environment: aws
+  archiveArtifacts: true
+  cwpVersion: 1.0
+  config:
+    bundle:
+      groupId: "io.jenkins.plugins.war"
+      artifactId: "artifact-manager-s3-war"
+      vendor: "Jenkins project"
+      title: "Jenkins WAR - Artifact Manager S3 Integration Tests"
+      description: "Demo build, which includes Artifact Manager S3 and plugins using ArtifactManager API for testing"
+    buildSettings:
+      version: 2.121.1
+      docker:
+        base: "jenkins/jenkins:2.121.1"
+        tag: "jenkins/artifact-manager-s3-war"
+        build: true
+        customSettings: '{"pushDockerImage": true, "dockerRegistry": "https://docker.cloudbees.com", "dockerRegistryCredentialsId": "80ca7cb9-b576-43df-9f54-ac49882dd7a9", "dockerTag": "jenkins/artifact-manager-s3-war"}'
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        version: 2.121.1
+    plugins:     
+      - groupId: "org.jenkins-ci.plugins.workflow"
+        artifactId: "workflow-api"
+        source:
+          version:  2.28
```

Comment:
This is already wrong. See #73.

IMO we should not merge anything that duplicates configuration specified in `pom.xml`.